SOFTWARE-5217: Update Major Package Release Policy (#7)
osg-htc/technology: docs/policy/community-testing.md

negative feedback after 7 calendar days.
In addition to the above requirements, major packages must also receive positive feedback and be approved by the Release
Manager.

> If a major package's functionality is tested by the OSG VM Universe tests, the release manager may approve that
> package for release without positive feedback after the package has been in testing for four weeks.

Reviewer: ```suggestion
If a major package has not received feedback after four weeks and it has been sufficiently tested by OSG integration
tests, the Release Manager may approve the package for release.
```

I think jamming the two conditions together in a single clause makes it easier to understand

If a package receives negative feedback, the offending package will be removed from the relevant testing repository.

Major Packages
--------------

The following packages are considered critical to the production Open Science Grid:

> - BLAHP

Reviewer: Should we keep this in since it's still a separate package in 3.6 release?

Author: I can go either way on this. We don't plan any more blahp releases and the integrated blahp is in wide used.

Author: Restored
